microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -11,16 +11,17 @@ ms.date: 02/11/2022
 # Debug WebView2 apps with Visual Studio Code
 
 Use Microsoft Visual Studio Code to debug scripts that run in WebView2 controls.  <!-- Make sure you're using Visual Studio Code version [insert build here] or later. -->
+Visual Studio Code has a built-in [debugger](https://code.visualstudio.com/docs/nodejs/browser-debugging) for browser debugging.
 
 
 <!-- ====================================================================== -->
 ## Create a launch.json file
 
-To debug your code, your project is required to have a `launch.json` file.  If your project doesn't have a `launch.json` file, create a new `launch.json` file and paste the following code into it:
+To debug your code, your project is required to have a `launch.json` file.  A `launch.json` file is a debugger configuration file to configure and customize the Visual Studio Code debugger. One of the properties needed to configure the debugger is the `request` property. There are two `request` types, `launch` and `attach`. The following code demonstrates launching the app from Visual Studio Code (rather than attaching the debugger to a running instance of the app). To do this, the app must have been built previously. If your project doesn't have a `launch.json` file, create a new `launch.json` file in the `.vscode` subfolder in your current project and paste the following code into it:
```

Comment:
```suggestion
To debug your code, your project is required to have a `launch.json` file.  A `launch.json` file is a debugger configuration file to configure and customize the Visual Studio Code debugger. One of the properties needed to configure the debugger is the `request` property. There are two `request` types, `launch` and `attach`.

The following code demonstrates launching the app from Visual Studio Code (rather than attaching the debugger to a running instance of the app). To do this, the app must have been built previously. If your project doesn't have a `launch.json` file, create a new `launch.json` file in the `.vscode` subfolder in your current project and paste the following code into it:
```
not sure if this breaking up of paragraph was considered before.  for increments of readable text.